### PR TITLE
Eflumerf/multiple streams

### PR DESCRIPTION
**Pull request by eflumerf**

### CODE REVIEW

File: test/config/df-segment.data.xml [hunk 1]
```diff
@@ -99,14 +99,51 @@
  <rel name="data_writer" class="DataWriterConf" id="dw-01"/>
  <rel name="uses" class="DFHWConf" id="dfhw-01"/>
 </obj>
+	
+	<obj class="DFApplication" id="df-02">
```
Comment (gcrone) [inline test/config/df-segment.data.xml:103]: Why are these lines indented differently to the other objects?

File: test/config/trigger-segment.data.xml [hunk 2]
```diff
@@ -116,7 +116,9 @@
  <attr name="application_name" type="string" val="daq_application"/>
  <rel name="runs_on" class="VirtualHost" id="vlocalhost"/>
  <rel name="contains">
-  <ref class="ReadoutGroup" id="dummyRog"/>
+  <ref class="ReadoutGroup" id="rog-1"/>
+	 <ref class="ReadoutGroup" id="rog-2"/>
+	 <ref class="ReadoutGroup" id="rog-3"/>
```
Comment (ArturSztuc) [inline test/config/trigger-segment.data.xml:121]: Something happened here with indentation

File: schema/appdal/trigger.schema.xml
Comment (ArturSztuc) [inline schema/appdal/trigger.schema.xml:1]: There's a lot of changes here that will take me a while to review. Is this all only just rearranging, or are there new additions/deletions here too? If it's the latter, then I will try to run this too.

**Review by gcrone — APPROVED** [review]
Looks OK to me

**Review by ArturSztuc — APPROVED** [review]
LGTM

File: test/config/df-segment.data.xml (hunk 1 above)
Comment (eflumerf) [reply test/config/df-segment.data.xml:103]: Likely due to a different editor. Let me do a dbe_main pass to clean it up.

**Pull request merged**